packages/es-components/webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  mode: 'production',
+  output: {
+    path: path.join(__dirname, 'bundle'),
+    filename: "[name].min.js"
+  },
+  externals: {
+    'react': 'react',
```

Comment:
I wonder if we're bundling too much. A quick `du -sh packages/es-components` shows `bundle` as being almost twice as big as the `es6` and `lib` builds. I've been trying to inspect `main.min.js` to see if react is still getting bundled in, but it's hard to tell. We may also need to consider adding `styled-components` to this if we want to avoid errors around having "multiple versions in the app" errors.

Reply:
I don't mind excluding all the peer dependencies, but even also excluding styled-components, the final bundle still ends up about twice the size of the other builds. I haven't been able to nail down exactly why that is, but my instinct is that it's Babel including a bunch of polyfills and boilerplate for es5 compatibility.